flow/src/main/java/com/sequenceiq/flow/repository/FlowChainLogRepository.java

```diff
@@ -33,6 +33,8 @@ public interface FlowChainLogRepository extends CrudRepository<FlowChainLog, Lon
 
     Optional<FlowChainLog> findFirstByFlowChainIdOrderByCreatedDesc(String flowChainId);
 
+    List<FlowChainLog> findByFlowChainIdOrderByCreatedDescIdDesc(String flowChainId);
```

Comment:
I think this should _replace_ the `findByFlowChainIdOrderByCreatedDesc` method

Reply:
I am sure of that. @lacikaaa what is your opinion?

Reply:
on the other hand `findFirstByFlowChainIdOrderByCreatedDesc` has the same bad behavior....

Reply:
Hi @votez , after talking with Kalyan and taking a look of the code even deeper, we changed where we want to update the code and we figured out why `created` are the same some time (in the description "CONTEXT"). I've already updated the code - I know `getCreated` anyway is a bug because `created` could be the same, but this PR is trying to fix the database backup issue we currently have so the scope is pretty limited. Please let me know if it looks good to you!!!=)